foxsec_sqs_worker.py

```diff
@@ -0,0 +1,231 @@
+"""FoxSec SQS worker"""
+import os
+import datetime
+import functools
+import time
+import json
+import ipaddress
+import boto3
+from botocore.exceptions import ClientError
+from pprint import pprint
+
+# Config
+# region = os.environ.get('AWS_DEFAULT_REGION', 'us-east-1')
+WAF_IPSET_ID = os.environ.get('IPSET_ID')
```

Comment:
It seems like all of these should be wrapped in an exception that quits the problem if they aren't set, with an explanation as to what is missing.